README.md

```diff
@@ -202,6 +203,87 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
 referrer="https://animixplay.to/"
 ```
 
+### Steam Deck
+
+#### Copypaste script:
+
+* Switch to Desktop mode (`STEAM` Button > Power > Switch to Desktop)
+* Open `Konsole` (Steam Deck Icon in bottom left corner > System > Konsole)
+* Copy the script, paste it in the CLI and press Enter("A" button on Steam Deck) 
+
+```
+git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf
+~/.fzf/install
+
+mkdir ~/.aria2c
+wget -O ~/.aria2c/aria2-1.36.0.tar.bz2 https://github.com/q3aql/aria2-static-builds/releases/download/v1.36.0/aria2-1.36.0-linux-gnu-64bit-build1.tar.bz2
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+mv ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.fzf/bin/
+chmod +x .fzf/bin/aria2c
+
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd ~ 
+cp ~/.ani-cli/ani-cli ~/.fzf/bin/
+chmod +x .fzf/bin/ani-cli
+
+flatpak install io.mpv.Mpv
+```
+press enter("A" button on Steam Deck) on questions
+
+#### Installation in steps:
+
+##### Install mpv (Flatpak version):
+
+```
+flatpak install io.mpv.Mpv
+```
+press enter("A" button on Steam Deck) on questions
+
+##### Install [fzf](https://github.com/junegunn/fzf): 
+
+```
+git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf
+~/.fzf/install
+```
+press enter("A" button on Steam Deck) on questions
+
+##### Install [aria2](https://github.com/aria2/aria2) (for --download):
+
+```
+mkdir ~/.aria2c
+wget -O ~/.aria2c/aria2-1.36.0.tar.bz2 https://github.com/q3aql/aria2-static-builds/releases/download/v1.36.0/aria2-1.36.0-linux-gnu-64bit-build1.tar.bz2
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+mv ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.fzf/bin/
+chmod +x .fzf/bin/aria2c
+```
```

Comment:
This should be emphasised to be a dependency for download only

Reply:
fzf installer puts it to ~/.fzf/bin and adds it to path
should i cp it to .local/bin too?

Reply:
> Also adding the install directory to the path would make sense, it may lead to some confusion (especially when troubleshooting). I may be wrong on this one, I don't know what the steamdeck "workflow" is like.

Im not quite understanding what you are trying to say with this line.

About workflow:
My position on the Copypaste install script is that it would **probably** be used by people who are not familiar with linux, so imo it should be as easy as possible.